src/app/utils/JournalUtils.js

```diff
@@ -6,6 +6,22 @@ import { Modules } from 'coriolis-data/dist';
 import { Modifications } from 'coriolis-data/dist';
 import { getBlueprint, setQualityCB } from './BlueprintFunctions';
 
+/**
+ * Check if an imported module is valid
+ * @param {Object} module the module to check
+ * @param {Object} moduleType the type of module to check
+ * @return {boolean} true if the module is valid
+ */
+function _isValidImportedModule(module, moduleType) {
+  // First of all, has the _moduleFromFdName function returned 'null'?
+  if (!module){
+    return false
+  }
+  else {
+    return true
+  }
+}
```

Comment:
Change to `return !module`

Reply:
Will do

Reply:
Cannot change to that, as the whole point of this function, is to convert the 'null' to 'false'. The upside of the if statement, is that it will return false if the variable is not set, if it is null, or if it is false, we then return the value of false to the function call in the various places that it's used throughout this file.